P0009/P0009.md

```diff
@@ -41,6 +41,9 @@ Special thanks to Tomasz Kaminski for invaluable help in preparing this paper fo
 
 - significant wording updates based on LWG feedback 
 - `submdspan` was moved into its own paper, allowing mdspan to go forward even if there is not enough time to review `submdspan`
+- Move wording change in paragraph 1 of [mdspan.extents.comp] from P2553 to P0009
+- Move wording change to [mdspan.layoutstride.ctor] (new precondition `other.stride(r) > 0` on `mapping(const StridedLayoutMapping&)` constructor) from P2553 to P0009
+- Per LWG small group request, change [mdspan.layout.reqmts] to account for `extents::size_type` possibly being negative (these changes could also go in P2553, but they are harmless to leave in P0009); this means that the layout mapping requirements will not permit negative strides for any strided layout in general, not just `layout_stride`
```

Comment:
Should these **[mdspan.layout.reqmts]** changes go in P2553?  They do actually depend on P2553 in order to be meaningful.

Reply:
Yeah, I think so.